scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -379,14 +379,30 @@ class FossIdTest : WordSpec({
                 .expectCreateScan(projectCode, scanCode, vcsInfo)
                 .expectDownload(scanCode)
                 .mockFiles(scanCode)
-            coEvery { service.runScan(USER, API_KEY, scanCode) } returns EntityResponseBody(status = 1)
+            coEvery {
+                service.runScan(
+                    USER, API_KEY,
+                    scanCode,
+                    *arrayOf(
+                        "auto_identification_detect_declaration" to "0",
+                        "auto_identification_detect_copyright" to "0"
+                    )
+                )
+            } returns EntityResponseBody(status = 1)
 
             val fossId = createFossId(config)
 
             fossId.scan(listOf(createPackage(createIdentifier(index = 1), vcsInfo)))
 
             coVerify {
-                service.runScan(USER, API_KEY, scanCode)
+                service.runScan(
+                    USER, API_KEY,
```

Comment:
I'm surprised that *detekt* doesn't complain here to put these arguments also into separate lines, like below. Would you mind still changing that?